satpy/readers/hrit_jma.py

```diff
@@ -283,6 +297,42 @@ def _mask_space(self, data):
         geomask = get_geostationary_mask(area=self.area)
         return data.where(geomask)
 
+    def _get_acq_time(self):
+        """
+        Acquisition times for a subset of scanlines are stored in the header
+        as follows:
+
+        b'LINE:=1\rTIME:=54365.022558\rLINE:=21\rTIME:=54365.022664\r...'
+
+        Missing timestamps in between are computed using linear interpolation.
+        """
+        buf_b = np.frombuffer(self.mda['image_observation_time'],
+                              dtype=image_observation_time)
+
+        # Replace \r by \n before encoding, otherwise encoding will drop all
+        # elements except the last one
+        buf_s = b''.join(buf_b['times']).replace(b'\r', b'\n').decode()
+
+        # Split into key:=value pairs; then extract line number and timestamp
+        splits = buf_s.strip().split('\n')
+        lines_sparse = [int(s.split(':=')[1]) for s in splits[0::2]]
+        times_sparse = [float(s.split(':=')[1]) for s in splits[1::2]]
+
+        if self.platform == HIMAWARI8:
```

Comment:
In case you wonder why only Himawari-8 is checked: The projection name in the header defines the platform, but at the moment it is identical for Himawari-8 and 9 (`GEOS(140.70)`). So there is no way to distinguish them.

Reply:
Great move, JMA...

Reply:
Isn't that because the design of the two platforms is that they aren't meant to be used (in operations) at the same time?

Reply:
That might be an explanation!